main/ui/src/main/java/org/cryptomator/ui/util/mount/LinuxGvfsWebDavMounter.java

```diff
@@ -70,4 +74,11 @@ public void unmount() throws CommandFailedException {
 		};
 	}
 
+	private Script openFMWithWebdavSchema(String schemaName, URI uri){
+		return Script.fromLines(
+				"set -x",
+				"xdg-open \""+schemaName+":$DAV_SSP\"")
```

Comment:
This [URI constructor](http://docs.oracle.com/javase/7/docs/api/java/net/URI.html#URI(java.lang.String,%20java.lang.String,%20java.lang.String)) may be feasible: `new URI("webdav", uri.getSchemeSpecificPath(), null)`